### ISSUE TITLE
[BUG] disappearing when 6/10

### ISSUE THREAD

**Issue by Dasbi** [guillaume009/runelite-plugin-action-progress#35]
when im doing zmi fletching broads at about 6/10 sometimes 5/10 it disappears. 

so when im running and fletching it seems to just disappear randomly, not when im standing still though

**Comment by guillaume009** [comment]
Can you elaborate on what you mean by zmi fletching? 

Moving the character acts as an interrupt hence its coded in a way so that the progress bar is hidden when the character do so.

Are you saying that it's possible to fletch while moving?

**Comment by Dasbi** [comment]
yes if you fletch headless arrows or any arrow you can start the action after you start moving, and then you fletch till you click for another action again
________________________________

**Comment by guillaume009** [comment]
Fix merged in Runelite. You should have it next time you load the plugin. Enjoy!
